pkg/kn/flags/podspec.go

```diff
@@ -163,3 +169,125 @@ func (p *PodSpecFlags) AddFlags(flagset *pflag.FlagSet) []string {
 	flagNames = append(flagNames, "user")
 	return flagNames
 }
+
+func (p *PodSpecFlags) ResolvePodSpec(cmd *cobra.Command, podSpec *corev1.PodSpec) error {
+	var err error
+
+	if cmd.Flags().Changed("env") {
+		envMap, err := util.MapFromArrayAllowingSingles(p.Env, "=")
+		if err != nil {
+			return fmt.Errorf("Invalid --env: %w", err)
+		}
+
+		envToRemove := util.ParseMinusSuffix(envMap)
+		err = UpdateEnvVars(podSpec, envMap, envToRemove)
+		if err != nil {
+			return err
+		}
+	}
+
+	if cmd.Flags().Changed("env-from") {
```

Comment:
This looks like code that exist in other commands?

Reply:
Removed. Thanks.